plugin.json

```diff
@@ -30,14 +30,14 @@
         "default": true
       },
       {
-        "key": "Secret",
+        "key": "secret",
         "display_name": "Webhook Secret:",
         "type": "generated",
         "help_text": "The secret used to authenticate the webhook to Mattermost.",
         "regenerate_help_text": "Regenerates the secret for the webhook URL endpoint. Regenerating the secret invalidates your existing Jira integrations."
       },
       {
-        "key": "StatsSecret",
+        "key": "stats_secret",
```

Comment:
Is this a breaking change?

Reply:
This _might_ affect those who were using the stats API, but to the best of my knowledge there aren't any monitors/consumers of it. It is a bug fix though, the secret was not actually being checked because of the mismatch.

Reply:
Are you sure the `json` tag in https://github.com/mattermost/mattermost-plugin-jira/blob/435868017e2c83cc160e23a1e880ac8fdabb0d84/server/plugin.go#L64 is used, if the config is set via the webapp e.g. by pressing the `Generate` button in the system console?

Reply:
Removed altogether in #836